src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/XamlSchemaContext.cs

```diff
@@ -646,7 +646,7 @@ internal XamlType GetXamlType(Type type, string alias)
             return (XamlValueConverter<TConverterBase>)result;
         }
 
-        internal virtual XamlMember GetProperty(PropertyInfo pi)
+        public virtual XamlMember GetProperty(PropertyInfo pi)
```

Comment:
All public API additions have to go through API review process first: https://github.com/dotnet/wpf/blob/master/Documentation/issue-guide.md#high-quality-features-and-api-suggestions

Reply:
Reverted changes.